.github/workflows/gc.yml

```
@@ -245,62 +245,45 @@ jobs:
           remove-codeql: "true"
           remove-docker-images: "true"
 
-      - name: Install Oras
-        run: |
-          # https://oras.land/docs/installation/#linux
-          VERSION="1.2.2"
-          curl -LO "https://github.com/oras-project/oras/releases/download/v${VERSION}/oras_${VERSION}_linux_amd64.tar.gz"
-          mkdir -p oras-install/
-          tar -zxf oras_${VERSION}_*.tar.gz -C oras-install/
-          sudo mv oras-install/oras /usr/local/bin/
-          rm -rf oras_${VERSION}_*.tar.gz oras-install/
-
-      - name: Pull ghcr.io/${{ inputs.repository }}:${{ inputs.tag }}
-        id: pull
-        run: |
-          MANIFEST=$(oras manifest fetch ghcr.io/${{ inputs.repository }}:${{ inputs.tag }})
-          DIGEST=$(echo ${MANIFEST} | jq -r '.layers.[] | select( .mediaType == "application/vnd.vulsio.vuls-data-db.dotgit.layer.v1.tar+zstd") | .digest')
-          TITLE=$(echo ${MANIFEST} | jq -r '.layers.[] | select( .mediaType == "application/vnd.vulsio.vuls-data-db.dotgit.layer.v1.tar+zstd") | .annotations."org.opencontainers.image.title"')
+      - name: Set up Go 1.x
+        uses: actions/setup-go@v6
+        with:
+          go-version: "stable"
 
-          oras blob fetch --output ${TITLE} ghcr.io/${{ inputs.repository }}:${{ inputs.tag }}@${DIGEST}
-          tar -xf ${TITLE}
-          rm ${TITLE}
+      - name: Install vuls-data-update
+        run: go install github.com/MaineK00n/vuls-data-update/cmd/vuls-data-update@main
 
-          BASENAME=$(basename "${TITLE}" .tar.zst)
-          echo "basename=${BASENAME}" >> ${GITHUB_OUTPUT}
+      - name: Pull ghcr.io/${{ inputs.repository }}:${{ inputs.tag }}
+        run: vuls-data-update dotgit pull --dir . --checkout main ghcr.io/${{ inputs.repository }}:${{ inputs.tag }}
 
       - name: Print git gc before .git size
         id: before
-        run: echo "size=$(du -sh ${{ steps.pull.outputs.basename }}/.git/ | cut -f1)" >> $GITHUB_OUTPUT
+        run: echo "size=$(du -sh ghcr.io/${{ inputs.repository }}:${{ inputs.tag }}/.git/ | cut -f1)" >> $GITHUB_OUTPUT
 
       - name: Set Git config
         run: |
-          git -C ${{ steps.pull.outputs.basename }} config gc.auto 0
-          git -C ${{ steps.pull.outputs.basename }} config pack.threads ${{ inputs.pack-threads }}
-          git -C ${{ steps.pull.outputs.basename }} config pack.windowMemory ${{ inputs.pack-windowMemory }}
+          git -C ghcr.io/${{ inputs.repository }}:${{ inputs.tag }} config gc.auto 0
+          git -C ghcr.io/${{ inputs.repository }}:${{ inputs.tag }} config pack.threads ${{ inputs.pack-threads }}
+          git -C ghcr.io/${{ inputs.repository }}:${{ inputs.tag }} config pack.windowMemory ${{ inputs.pack-windowMemory }}
```

If the above comment is right, the same here:

```suggestion
          git -C ghcr.io/${{ inputs.repository }}/${{ inputs.tag }} config gc.auto 0
          git -C ghcr.io/${{ inputs.repository }}/${{ inputs.tag }} config pack.threads ${{ inputs.pack-threads }}
          git -C ghcr.io/${{ inputs.repository }}/${{ inputs.tag }} config pack.windowMemory ${{ inputs.pack-windowMemory }}
```

---

fixed: https://github.com/vulsio/vuls-data-db/compare/dd8769083a3886925b73d1566105018e4fd354d0..41980d1d9dc42dfdac44f16aaf0e692aca97693d